lightning/src/ln/channelmanager.rs

```diff
@@ -8518,6 +8539,16 @@ where
 							update: update.clone(),
 						});
 				}
+				if $chan_in_flight_upds.is_empty() {
```

Comment:
Should we check if any updates were actually removed before generating the background event?